### PR TITLE
Feat: practice round pre instruction

### PR DESCRIPTION
**Pull request by hardisp**
This PR contain new feature instruction pre practice regarding client's feedback

- [x] add pre practice instruction page

### CODE REVIEW

**Review by hubertusputu1 — APPROVED** [review]
LGTM. Tested.

**Comment by npaton** [comment]
This should also be before phase 2!

**Comment by npaton** [comment]
So before step 2 (when we show AI results), there's also a practice, so this should be shown then too. At that point, it should be added as a Round, like the step 2 instructions/question, just right after them actually.

**Comment by hardisp** [comment]
@npaton It's already on there. I've added pre practice page before the practice both in step one and step two.

File: client/game/component/Instruction.jsx [hunk 1]
```diff
@@ -32,14 +39,22 @@ export default class Instruction extends React.Component {
...
   render() {
     const {
       game: { treatment },
     } = this.props;
     const { interpretationType = "None", giveFeedback } = treatment;
-    const { toast, quiz } = this.state;
+    const { toast, quiz, preRound } = this.state;
+
+    if (preRound) {
+      return <PreRound onNext={this.handleToNextRound} step={2} />;
```
Comment (hardisp) [inline client/game/component/Instruction.jsx:54]: @npaton I added here

**Comment by npaton** [comment]
Ok, my bad, thank you!

**Pull request merged**